wp-content/plugins/hurumap/src/FlourishChart.js

```diff
@@ -91,21 +84,25 @@ function FlourishChart({ chart, onChange, onDelete }) {
     multiple: false
   });
 
+  const [title, setTitle] = useState(chart.title);
+
   return (
     <Paper>
       <Grid container className={classes.root} spacing={2}>
         <Grid item md={4}>
           <Grid container direction="column" spacing={2}>
             <Grid item>
               <TextField
+                fullWidth
                 label="Title"
                 type="text"
-                value={chart.title}
+                value={title}
                 InputLabelProps={{
                   shrink: true
                 }}
-                fullWidth
+                onChange={e => setTitle(e.target.value)}
```

Comment:
Wanted to push `onFocus` for this together with `onBlur`. But this will do too

Reply:
let update this line too to use state title https://github.com/TakwimuAfrica/Dashboard/blob/2f7ab6730c56312092235f11b4fa179ff4471d71/wp-content/plugins/hurumap/src/FlourishChart.js#L209